assets/xml/objects/object_dnt.xml

```diff
@@ -1,91 +1,106 @@
+        <DList Name="gBusinessScrubLeftThighDL" Offset="0x9838" />
+        <DList Name="gBusinessScrubRightFootDL" Offset="0x98D8" />
+        <DList Name="gBusinessScrubRightShinDL" Offset="0x99E8" />
+        <DList Name="gBusinessScrubRightThighDL" Offset="0x9AE0" />
+
+        <!-- Textures -->
+        <Texture Name="gBusinessScrubSkinTex" OutName="business_scrub_skin" Format="rgba16" Width="16" Height="16" Offset="0x9B80" />
+        <Texture Name="gBusinessScrubLeafTex" OutName="business_scrub_leaf" Format="rgba16" Width="16" Height="16" Offset="0x9D80" />
+        <Texture Name="gBusinessScrubBagOpeningTex" OutName="business_scrub_bag_opening" Format="rgba16" Width="8" Height="16" Offset="0x9F80" />
+        <Texture Name="gBusinessScrubBagCordTex" OutName="business_scrub_bag_cord" Format="rgba16" Width="8" Height="16" Offset="0xA080" />
+        <Texture Name="gBusinessScrubBagTex" OutName="business_scrub_bag" Format="rgba16" Width="2" Height="16" Offset="0xA180" />
+        <Texture Name="gBusinessScrubHatTex" OutName="business_scrub_hat" Format="rgba16" Width="8" Height="8" Offset="0xA1C0" />
+        <Texture Name="gBusinessScrubHatLeafTex" OutName="business_scrub_hat_leaf" Format="rgba16" Width="8" Height="16" Offset="0xA240" />
+        <Texture Name="gBusinessScrubHairTex" OutName="business_scrub_hair" Format="rgba16" Width="16" Height="32" Offset="0xA340" />
+        <Texture Name="gBusinessScrubScalpTex" OutName="business_scrub_scalp" Format="rgba16" Width="8" Height="8" Offset="0xA740" />
+        <Texture Name="gBusinessScrubSnoutTex" OutName="business_scrub_snout" Format="rgba16" Width="16" Height="8" Offset="0xA7C0" />
+        <Texture Name="gBusinessScrubEyeTex" OutName="business_scrub_eye" Format="rgba16" Width="16" Height="16" Offset="0xA8C0" />
+
+        <!-- Skeleton -->
+        <Limb Name="gBusinessScrubRootLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_ROOT" Offset="0xAAC0" />
+        <Limb Name="gBusinessScrubBodyLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_BODY" Offset="0xAACC" />
+        <Limb Name="gBusinessScrubRightThighLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_RIGHT_THIGH" Offset="0xAAD8" />
+        <Limb Name="gBusinessScrubRightShinLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_RIGHT_SHIN" Offset="0xAAE4" />
+        <Limb Name="gBusinessScrubRightFootLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_RIGHT_FOOT" Offset="0xAAF0" />
+        <Limb Name="gBusinessScrubLeftThighLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_LEFT_THIGH" Offset="0xAAFC" />
+        <Limb Name="gBusinessScrubLeftShinLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_LEFT_SHIN" Offset="0xAB08" />
+        <Limb Name="gBusinessScrubLeftFootLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_LEFT_FOOT" Offset="0xAB14" />
+        <Limb Name="gBusinessScrubFaceLimb" Type="Standard" EnumName="BUSINESS_SCRUB_LIMB_FACE" Offset="0xAB20" />
```

Comment:
I know this guy has some weird anatomy, but the use of "face" here is weird when it wraps around his whole body:
![image](https://github.com/zeldaret/mm/assets/12245827/871407c5-a11e-44fd-aa84-230d4880d461)

Maybe just call this one "head"? Or, if you don't want to use that term, call them "upper body" and "lower body"?